presto-server-rpm/src/main/rpm/preinstall

```diff
@@ -64,10 +64,10 @@ if [ "$java_found" = false ]; then
 +======================================================================+
 |      Error: Required Java version could not be found                 |
 +----------------------------------------------------------------------+
-| Please download the latest Oracle JDK/JRE from the Java web site     |
-|       > http://www.oracle.com/technetwork/java/javase/downloads <    |
+| Please download the latest OpenJDK or Oracle JDK/JRE.                |
```

Comment:
Saying "latest" is misleading here, since that would be Java 14. Let's change this to
```
| Please install JDK 11. On RHEL/CentOS, use java-11-openjdk-devel.    |
|                                                                      |
| You can also download an OpenJDK 11 build, such as Zulu Community:   |
|   https://www.azul.com/downloads/zulu-community/                     |

Reply:
Updated